### PR TITLE
Feature/117 toaster cookies

### PR DESCRIPTION
**Pull request by BogAlex4ndr**
Created Cookie toaster, added key in local storage that user sees it only when he enter website first time

### CODE REVIEW

**Review by vnko1 — APPROVED** [review]

**Review by ArtemBohak — APPROVED** [review]
Everything seems fine, you can move the toaster to the /auth/register page

File: src/components/CookiesToaster/CookiesToaster.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  useEffect(() => {
+    const acceptedCookies = localStorage.getItem("CookieAccept");
+    return () => {
+      if (acceptedCookies === "1") {
+        setToasterHide(true);
+      }
+    };
+  }, []);
+
+  const CookieAccept = () => {
```
Comment (vasilkovalyov) [inline src/components/CookiesToaster/CookiesToaster.tsx:18]: name of methods start from loverCase - cookieAccept

File: src/components/CookiesToaster/CookiesToaster.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <div className={styles["info-block"]}>
+        <h5 className={styles["info-block__title"]}>
+          This website uses cookies
+        </h5>
+        <p className={styles["info-block__text"]}>
+          We use cookies to make sure this website can function. By continuing
+          to browse on this website, you agree to our use of cookies.
+        </p>
+
+        <div onClick={CookieAccept} className={styles["info-block__accept"]}>
```
Comment (vasilkovalyov) [inline src/components/CookiesToaster/CookiesToaster.tsx:33]: it is interactive element, make it like a button

Comment (BogAlex4ndr) [reply src/components/CookiesToaster/CookiesToaster.tsx:33]: fixed

File: src/components/CookiesToaster/CookiesToaster.tsx (hunk 1 above)
Comment (BogAlex4ndr) [reply src/components/CookiesToaster/CookiesToaster.tsx:18]: fixed

**Pull request merged**